### ISSUE TITLE
requestIdleCallback

### ISSUE THREAD

**Issue by myakura** [web-platform-tests/interop#563]
### Description

`requestIdleCallback()` allows devs to schedule callbacks to be run in an idle time.

### Specification

https://w3c.github.io/requestidlecallback/

### Open Issues

https://github.com/w3c/requestidlecallback/issues

There may be some other issues. The spec once reached Proposed Recommendation in 2017 (!!) but somehow stalled in the stage, then went back to WD in 2022.
https://www.w3.org/standards/history/requestidlecallback/

### Tests

https://wpt.fyi/results/requestidlecallback

### Current Implementations

- [X] Blink
- [X] Gecko
- [X] WebKit

### Standards Positions

https://chromestatus.com/feature/5572795866021888

### Browser bug reports

https://bugs.webkit.org/show_bug.cgi?id=164193
https://bugzilla.mozilla.org/show_bug.cgi?id=1198381

### Developer discussions

https://twitter.com/ElliottZ/status/1574538125363351552

### Other

With WebKit implementing the API this year and browsers passing most of the tests, we may not need to put it in the focus area. I'm filing it because there were devs asking for it last year.

**Comment by jgraham** [comment]
Seems like there's 1/2 remaining test failures here. Do we have evidence that those are cases that are affecting developers? If so this could be worthwhile, grouped with some other proposals, if not it may not meet the bar of representing a real interop issue.

**Comment by dandclark** [comment]
Thank you for proposing requestIdleCallback for inclusion in Interop 2024.

We wanted to let you know that this proposal was not selected to be part of Interop this year.

This proposal was not included in Interop 2024 because when we looked at its Web Platform Tests results, we found that it is already largely interoperable between tested browsers and does not need special coordinated attention.

For an overview of our process, see [proposal selection](https://github.com/web-platform-tests/interop/blob/main/2024/selection-process.md). Thank you again for contributing to Interop 2024!

_Posted on behalf of the Interop team._
